recipes/ecell4_base/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "ecell4_base" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ecell/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 2e60e21e82c240739259c8a246c8889b9be86800420930eed39ba947d88ba7bc
+
+build:
+  number: 0
+  script_env:
+    - CC
+    - CXX
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python
+    - setuptools
+    - pybind11
+    - cmake
+    - hdf5
+    - gsl
+    - boost
+    - blas
+
+  run:
+    - libgcc-ng  # [linux]
+    - libstdcxx-ng  # [linux]
+    - gsl
+    - hdf5
+    - zlib
+    - python
+    - vs2015_runtime  # [win]
+
+test:
+  imports:
+    - ecell4_base.core
+    - ecell4_base.bd
+    - ecell4_base.egfrd
+    - ecell4_base.gillespie
+    - ecell4_base.meso
+    - ecell4_base.ode
+    - ecell4_base.sgfrd
+    - ecell4_base.spatiocyte
+
+about:
+  home: https://github.com/ecell/ecell4_base
+  license: GPL 3.0
```

Could you please change it to:
```suggestion
  license: GPL-3.0
  license_family: GPL
```